tests/test_storage_manager_advanced.py

```
@@ -0,0 +1,342 @@
+                    self.assertIsNotNone(result.file_path)
+
+    @patch("subprocess.run")
+    def test_git_stash_operations(self, mock_run):
+        """Git stash操作のテスト"""
+        git_handler = GitHandler(auto_commit=False)
+
+        # Arrange
+        mock_run.return_value = Mock(returncode=0, stdout="Saved working directory")
+
+        # Act - stash操作をシミュレート
+        success, _ = git_handler.add_files(["*.csv"])
```

Ruff (0.14.2)

Unpacked variable `success` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)